src/components/Book.js

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import styles from '../styles/book.module.css';
 import { removeBookAPI } from '../redux/features/bookstore/BooksSlice';
+import Button from './Button';
+import progressImg from '../assets/progressBar.png';
 
 function Book({ id, title, author }) {
```

Comment:
- [ ] Currently, there is a gray border present in the Zeplin design that is absent in yours so kindly include that as well 👍 
- [ ] From the images below, you can see that the size of your card is a little smaller than that of the Zeplin design. So kindly fix that 👍 
- [ ] Kindly ensure that the `author name`, `remove`, `comment`, `edit` and  `completed` buttons/texts have a similar font size, font-weight and color as seen in the Zeplin design 😉 
- Your Design
![image](https://github.com/Estete9/bookstore-react/assets/102699958/3d73c33d-4059-4c6a-ad2f-c3b8750a2612)

- The Zeplin Design
![image](https://github.com/Estete9/bookstore-react/assets/102699958/0532b098-e4e3-4ebd-a732-76148b120bd7)